savoirfairelinux/sous-chef
# fix529 Pick Sides Ingredients separately

## Fixes #529.

### Changes proposed in this pull request:

* delivery / fixtures / sample_data : added one component having component group = "sides" for use in tests.
* delivery / forms , delivery / templates / ingredients : new field and validation
* delivery / views : handle Sides ingredients, save them in today's menu (for clash detection to be added in the meal labels generation - this will be covered by fixing Issue #531.)
* meal / models : new component group choice = 'sides'.
* order / models : in order creation, skip component group choice = 'sides'. 
* src / meallabels.pdf : delete file such that it will not be tracked by git anymore.

### Status


### How to verify this change

* In Django Admin, create a new Component named "Day s Sides" and set its component group = "Sides". (Do this only once).
* In Kitchen Count step 2, choose Sides ingredients, click "Next - Print Kitchen Count", go in Django Admin look for the Component names "Day s Sides", see its Component Ingredients for today's date.
